polls-2024-03-25 (#2599)
makerdao/community: governance/polls/SparkLend Ethereum - Increase WBTC supply cap max parameter to 10k - March 25, 2024.md

---

> title: SparkLend Ethereum - Increase WBTC supply cap max parameter to 10k - March 25, 2024

Reviewer: :yellow_circle: **minor**: capitals in title

Make consistent with prior poll https://vote.makerdao.com/polling/QmYYoAMe

```suggestion
title: SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 25, 2024
```

Reviewer: capitals in file name if updating

> summary: Signal your support or opposition to increasing the WBTC supply cap max parameter to 10k WBTC

Reviewer: :green_circle: **nit**: see above

```suggestion
summary: Signal your support or opposition to increasing the WBTC supply cap max parameter to 10,000 WBTC
```

discussion_link: https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-03-25T16:00:00
end_date: 2024-03-28T16:00:00
---

> # Poll: SparkLend Ethereum - Increase WBTC supply cap max parameter to 10k - March 25, 2024

Reviewer: See line 2 comment

```suggestion
# Poll: SparkLend Ethereum - Increase WBTC Supply Cap Maximum - March 25, 2024
```

The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 25 at 16:00 UTC.

**This is a binary vote.**
- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

## Review

The community can vote in this poll to express support or opposition to the following SparkLend [CapAutomator](https://github.com/marsfoundation/sparklend-cap-automator) change:

> - Increase [WBTC market](https://app.spark.fi/reserve-overview/?underlyingAsset=0x2260fac5e5542a773aa44fbcfedf7c193bc2c599&marketName=proto_spark_v3) supply cap `max` parameter by 5,000 WBTC from 5,000 WBTC to **10,000 WBTC**

Reviewer: :red_circle: **blocking**

"Current" value is incorrect. 6k has been approved and will be included in the forthcoming spell (https://github.com/makerdao/community/pull/2597/files#diff-46eac166654845e1c858fca919ba2d2ec7d5114fdc7248747924e2fc3b8b8e14R137)

```suggestion
- Increase [WBTC market](https://app.spark.fi/reserve-overview/?underlyingAsset=0x2260fac5e5542a773aa44fbcfedf7c193bc2c599&marketName=proto_spark_v3) supply cap `max` parameter by 4,000 WBTC from 6,000 WBTC to **10,000 WBTC**
```
